blocked-edges/4.17.0-OVNLocalnetWithNoSubnets.yaml

```diff
@@ -0,0 +1,14 @@
+to: 4.17.0
+from: ^4[.]16[.].*
+url: https://issues.redhat.com/browse/SDN-5485
+name: OVNLocalnetWithNoSubnets
+message: |-
+  Clusters with localnet topology networks with no subnets defined (configuration typical for OpenShift Virtualization)
+  would fail to update and create new workloads.
+matchingRules:
+  - type: PromQL
+    promql:
+      promql: |
+        group(csv_succeeded{name=~".*hyperconverged.*"})
```

Comment:
nit, missing the `_id=""` HyperShift placeholder (5cb2e937286, #3591).

Reply:
I also think we can tighten the operator-name matching by using `kubevirt-hyperconverged-operator[.].*` as the regular expression.  I'm not clear on why folks had been floating the less-specific `.*hyperconverged.*`.

Reply:
yeah I wondered about the name too, then just assumed @sradco knows the domain better than I do and copied what she wrote. But I do not mind expanding it, we can refine if we discover the operator changed its name over time or something. I'll fix the `_id`, I swear I had it there but then I think I badly pasted some bit and overwrote it.

Reply:
Name looks reliable unless RH messes with it in OpenShift Virtualization:

```
$ for csv in (find deploy/olm-catalog/ -name '*.clusterserviceversion.yaml')
    yq .metadata.name < $csv
end
kubevirt-hyperconverged-operator.v0.0.1
kubevirt-hyperconverged-operator.v0.0.2
kubevirt-hyperconverged-operator.v0.0.3
kubevirt-hyperconverged-operator.v1.0.0
kubevirt-hyperconverged-operator.v1.1.0
kubevirt-hyperconverged-operator.v1.10.0
kubevirt-hyperconverged-operator.v1.11.0
kubevirt-hyperconverged-operator.v1.12.0
kubevirt-hyperconverged-operator.v1.13.0
kubevirt-hyperconverged-operator.v1.14.0
kubevirt-hyperconverged-operator.v1.2.0
kubevirt-hyperconverged-operator.v1.3.0
kubevirt-hyperconverged-operator.v1.4.0
kubevirt-hyperconverged-operator.v1.5.0
kubevirt-hyperconverged-operator.v1.6.0
kubevirt-hyperconverged-operator.v1.7.0
kubevirt-hyperconverged-operator.v1.8.0
kubevirt-hyperconverged-operator.v1.9.0
```